jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -177,19 +177,23 @@ pipeline {
                             postCleanup()
                         }
                         aborted {
-                            script {
-                                def stackName = "test-single-security-${BUILD_ID}-${ARCHITECTURE}-perf-test"
+                             script {
+                                stackName = "test-single-security-${BUILD_ID}-${ARCHITECTURE}-${BUILD_NUMBER}"
                                 withCredentials([string(credentialsId: 'jenkins-aws-account-public', variable: 'AWS_ACCOUNT_PUBLIC')]) {
-                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session', region:'eu-west-1') {
                                         try {
-                                            def stack = cfnDescribe(stack: stackName)
+                                            def stack = null
+                                            try {
+                                                stack = cfnDescribe(stack: stackName)
+                                            } catch (Exception e) {
+                                                echo "Stack '${stackName}' does not exist, nothing to remove"
+                                            }
                                             if (stack != null) {
```

Comment:
As discussed I don't think it will ever be null. It will either return a stack status or stack not found neither is `null` Were you able to test this?

Reply:
Thanks for the reviewing and commenting, @gaiksaya ..  yes, I redo the test on staging and confirm there will be an exception happened when the stack used in cfnDescribe does not exist; hence, I tried the try-catch, and it did help on this situation. :)
When the stack does not exist, the cfnDescribe throw exception, and then, the catch block will be able to eacho out the stack does not exit.